### PR TITLE
Adding functions for getting device io max size and hw io size.

### PR DESCRIPTION
**Pull request by karolinavelkaja**
adding functions for getting device io max size and hw io size. Also returning Size() from property.

Signed-off-by: Karolina Rogowska <karolina.rogowska@intel.com>

### CODE REVIEW

**Review by Ostrokrzew — DISMISSED** [review]

File: storage_devices/device.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ def get_io_stats(self):
 
     def get_sysfs_property(self, property_name):
         path = os.path.join(disk_utils.get_sysfs_path(self.get_device_id()), "queue", property_name)
-        return TestRun.executor.run_expect_success(f"cat {path}").stdout
+        return Size(int(TestRun.executor.run_expect_success(f"cat {path}").stdout), Unit.KiloByte)
```
Comment (Deixx) [inline storage_devices/device.py:70]: Sysfs properties are not only sizes, you cannot do this for all of them

**Review by Deixx — CHANGES_REQUESTED** [review]

File: storage_devices/device.py (hunk 1 above)
Comment (Deixx) [reply storage_devices/device.py:70]: Additionally those are more likely kibibytes not kilobytes

Comment (karolinavelkaja) [reply storage_devices/device.py:70]: Correct, will fix this! Thanks!

File: storage_devices/device.py [hunk 2]
```diff
@@ -67,7 +67,7 @@ def get_io_stats(self):
 
     def get_sysfs_property(self, property_name):
         path = os.path.join(disk_utils.get_sysfs_path(self.get_device_id()), "queue", property_name)
-        return TestRun.executor.run_expect_success(f"cat {path}").stdout
+        return int(TestRun.executor.run_expect_success(f"cat {path}").stdout)
```
Comment (Ostrokrzew) [inline storage_devices/device.py:70]: You cannot cast output to `int` here - not all sysfs properties are numbers.

**Review by Ostrokrzew — CHANGES_REQUESTED** [review]

File: storage_devices/device.py (hunk 2 above)
Comment (karolinavelkaja) [reply storage_devices/device.py:70]: Oh ok, I've checked the usages of all properties here and I thought that all were numbers but maybe some of them which I've got zeroed may be set to other values. Thank you, I will fix this! :)

**Review by Ostrokrzew — APPROVED** [review]

**Review by Deixx — APPROVED** [review]

**Review by katlapinka — APPROVED** [review]

**Pull request merged**